README.md

```diff
@@ -109,43 +109,15 @@ Your package should have this structure:
 | foo-tests.ts | This contains sample code which tests the typings. This code does *not* run, but it is type-checked. |
 | tsconfig.json | This allows you to run `tsc` within the package. |
 
-`index.d.ts` should start with a header looking like:
+Generate these by running the following from the root directory:
 
-```ts
-// Type definitions for foo 1.2
-// Project: https://github.com/baz/foo
-// Definitions by: My Self <https://github.com/me>
-// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-```
-
-The `Project` link does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.
-
-`tsconfig.json` should look like this:
-
-```json
-{
-    "compilerOptions": {
-        "module": "commonjs",
-        "target": "es6",
-        "noImplicitAny": true,
-        "strictNullChecks": false,
-        "baseUrl": "../",
-        "typeRoots": [
-            "../"
-        ],
-        "types": [],
-        "noEmit": true,
-        "forceConsistentCasingInFileNames": true
-    },
-    "files": [
-        "index.d.ts",
-        "foo-tests.ts"
-    ]
-}
+```sh
+npm install
+npm run compile-scripts
+node scripts/new-package.js new-package-name
```

Comment:
consider checking in the .js file and making the script an npm script. e.g. `npm run new-package new-package-name`